src/xcode/ENA/ENA.xcodeproj/xcshareddata/xcschemes/ENA.xcscheme

```diff
@@ -114,7 +114,6 @@
       buildConfiguration = "Debug"
       selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
       selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
-      language = "en"
```

Comment:
Was this a conscious Decision?

Reply:
Yes. I don't think the App should start in English every time. Or do you know that somebody set this to English for everyone on purpose?

Reply:
Just checking, sometimes it happens to me that I change something in the scheme and forget to revert it :D